patches/chrome-installer-linux-debian-build.sh.patch

```diff
@@ -38,27 +38,34 @@ index 306b59c138b7cb5df21ad6e9d502bb7a46a104c6..9e6da48138b26754e9e5b000451e65db
        grep '^ Depends: ' | sed 's/^ Depends: //' | sed 's/, /\n/g' | \
        LANG=C sort > "${ACTUAL_DEPENDS}"
    BAD_DIFF=0
-@@ -157,8 +157,11 @@ verify_channel() {
+@@ -156,13 +156,18 @@ verify_channel() {
+       RELEASENOTES="https://chromereleases.googleblog.com/search/label/Stable%20updates"
        ;;
-     unstable|dev|alpha )
+     beta|testing )
++
```

Comment:
never add empty lines in patches

Reply:
Wow, yes, not sure what happened here.

Reply:
Fixed in 14148ce6c7e3616560ae3c67c033de93a008893c